### PR TITLE
Add Json.toBytes to convert a JsValue directly to a byte array

### PR DESCRIPTION
**Pull request by gmethvin**
See also playframework/playframework#6334

### CODE REVIEW

**Comment by marcospereira** [comment]
Should this be also added to the Java API?

**Comment by gmethvin** [comment]
@marcospereira Actually, we might not even want to keep `play.libs.Json` in the `play-json` project, since it's only a thin wrapper for Jackson. I just moved it because it was in the play-json module already, but it probably makes more sense to have it in `playframework` since it's not really a library on its own and has nothing to do with the Scala JSON library.

**Comment by marcospereira** [comment]
Ok, will you add the `Writeable` change here?

**Comment by gmethvin** [comment]
`Writeable` is a type class used in Play Scala actions to automatically convert types to an `HttpEntity` to be sent as part of a `Result`. It doesn't make sense to include here since that's the responsibility of the main Play library, not play-json.

File: play-json/src/main/scala/play/api/libs/json/Json.scala [hunk 1]
```diff
@@ -60,6 +60,14 @@ object Json {
    */
   def stringify(json: JsValue): String = JacksonJson.generateFromJsValue(json)
 
+  /**
+    * Convert a JsValue directly to an array of bytes (using UTF-8 encoding)
```
Comment (cchantep) [inline play-json/src/main/scala/play/api/libs/json/Json.scala]: Indent issue? (https://github.com/playframework/play-json/pull/18/commits/aa1776a37d1ad56be584fb26b7b94e4f1c1e351f)

**Comment by gmethvin** [comment]
@cchantep I think this is good now. Afterward we could release M2 and update #6334.

**Comment by cchantep** [comment]
Would be nice to also have #13 in a next release.

**Pull request merged**